### PR TITLE
feat(hooks,cli): add scheduler admin route + cli

### PR DESCRIPTION
**Pull request by sguiheux**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

File: cli/cdsctl/admin_hooks_scheduler.go [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+		cli.NewListCommand(adminHooksSchedulersListAllCmd, adminHooksSchedulersListAllRun, nil, withAllCommandModifiers()...),
+		cli.NewGetCommand(adminHooksGetSchedulerCmd, adminHooksGetSchedulerRun, nil, withAllCommandModifiers()...),
+		cli.NewDeleteCommand(adminHookSchedulerDeleteCmd, adminHookSchedulerDeleteRun, nil),
+	})
+}
+
+var adminHookSchedulerDeleteCmd = cli.Command{
+	Name:    "delete",
+	Aliases: []string{"rm", "remove", "del"},
+	Short:   "Delete a specific scheduler by his identifier",
```
Comment (fsamin) [inline cli/cdsctl/admin_hooks_scheduler.go:31]:
```suggestion
	Short:   "Delete a scheduler by its identifier",
```

File: cli/cdsctl/admin_hooks_scheduler.go [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+		}
+	}
+
+	return nil
+}
+
+var adminHooksGetSchedulerCmd = cli.Command{
+	Name:    "get",
+	Aliases: []string{"show"},
+	Short:   "Get a scheduler by his identifier",
```
Comment (fsamin) [inline cli/cdsctl/admin_hooks_scheduler.go:72]:
```suggestion
	Short:   "Get a scheduler by its identifier",
```

**Review by fsamin — CHANGES_REQUESTED** [review]

File: cli/cdsctl/admin_hooks_scheduler.go [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+	"time"
+
+	"github.com/ovh/cds/cli"
+	"github.com/ovh/cds/sdk"
+	"github.com/spf13/cobra"
+)
+
+var adminHooksSchedulerCmd = cli.Command{
+	Name:    "scheduler",
+	Aliases: []string{"s", "schedule", "scheudlers"},
```
Comment (yesnault) [inline cli/cdsctl/admin_hooks_scheduler.go:16]:
```suggestion
	Aliases: []string{"s", "schedule", "schedulers"},
```

File: cli/cdsctl/admin_hooks_scheduler.go [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+	Short:   "Manage repositories where there were events",
```
Comment (yesnault) [inline cli/cdsctl/admin_hooks_scheduler.go:17]:
```suggestion
	Short:   "Manage workflow schedulers",
```

**Review by yesnault — CHANGES_REQUESTED** [review]

**Review by yesnault — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-cds#20852.0 ✔ 
* 
  * Build and Package All -   
  * Build and Package Contrib Action ✔ 
  * Build and Package Contrib Code Plugin ✔ 
  * Build and Package Contrib Integrations ✔ 
  * Build and Package Engine, Worker, CLI and Doc ✔ 
  * Build and Package UI ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20852.0 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

**Review by richardlt — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-cds#20871.0 ✔ 
* 
  * Build and Package All -   
  * Build and Package Contrib Action ✔ 
  * Build and Package Contrib Code Plugin ✔ 
  * Build and Package Contrib Integrations ✔ 
  * Build and Package Engine, Worker, CLI and Doc ✔ 
  * Build and Package UI ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20871.0 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

**Review by fsamin — APPROVED** [review]

**Pull request merged**